src/components/study-container.js

```diff
@@ -625,8 +626,14 @@ export function StudyContainer({ view, onChangeTab }) {
         // if only node renaming, do not reload network
         if (isNodeRenamed(previousCurrentNode, currentNode)) return;
         if (!isNodeBuilt(currentNode)) return;
-        loadNetwork(true);
-    }, [loadNetwork, currentNode, wsConnected]);
+        if (
+            !network ||
```

Comment:
If first time and network is undefined then why not calling `loadNetwork(false)`
There is no case where we call this ( `loadNetwork(false)` or `loadNetwork()`), maybe there is something we are missing...

Reply:
I think we moved away from the parameter name and comment of loadNetwork function.
Currently, we do the network initialization with a loadNetwork(isUpdate=true).
A clean up can be done in this PR, we'll talk about it